veles/secrets/onepasswordkeys/validator.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package onepasswordkeys provides validation for 1Password Service Account Tokens.
+package onepasswordkeys
+
+import (
+	"context"
+
+	"github.com/1password/onepassword-sdk-go"
```

Comment:
It's difficult to import new libraries internally (and also makes it difficult to test) so we'd prefer to send the network calls directly. I assume the SDK just creates an HTTP client and calls a specific endpoint? In that case we can do the same and send the appropriate request to list vaults through a raw HTTP client.

Reply:
Sounds like validation is not feasible at the moment without the SDK. I'd say let's drop the validation part from the PR in that case.

Adding an extractor for connect server access tokens sounds good to me. Are other 1password secrets (secret keys and service tokens) also stored in specific files such as 1password-credentials.json? In that case I'd prefer we scan for them through an extractor with file path filters as well since the regex for at least the secret keys are false positive prone otherwise.